lib/auto-build.ts

```diff
@@ -29,6 +29,15 @@ export interface AutoBuildOptions {
    */
   readonly publicLogs?: boolean;
 
+  /**
+   * Whether to publish a link to build logs when build is successful.
+   *
+   * @see https://github.com/jlhood/github-codebuild-logs#app-parameters
+   *
+   * @default false
+   */
+  readonly publicLogsOnSuccess?: boolean;
```

Comment:
Let's make the default true. I find it very useful to get an email when my build passes.

Reply:
😄 this is exactly the kind of noise I was referring to in the PR description 😄 

I personally don't like to get 5 emails when I push 5 different commits to a PR... and I like to only get alerts/notifications when something fails (this is the behavior of failed GitHub workflows).

Tell me if you still want to change this default, I will of course adapt accordingly 😉

Reply:
Let's change the default to true